### ISSUE TITLE
Add support for https://mangalivre.net/

### ISSUE THREAD

**Issue by Kuraigus** [HDoujinDownloader/HDoujinDownloader#59]
Could you please add support for MangaLivre: https://mangalivre.net/

MangaLivre its a very complete source for manga/manwha in PT-BR language, would love to see if its possible to have mangalivre in HDoujin!

**Comment by HDoujinDownloader** [comment]
I'm getting an "Error 1020" page from Cloudflare when I try to access this site. It's probable that they're using Cloudflare's firewall rules to restrict what IPs can access it. This seems to be a common occurrence with PT-BR manga websites for some reason.

I assume you're accessing it from Brazil?

**Comment by Kuraigus** [comment]
Yeah, Im acessing with an Brazilian IP, is there anything I could do to help you out?

**Comment by HDoujinDownloader** [comment]
No worries, it looks like the VPN I use has servers in Brazil. I'll report back when I get the opportunity to test it out.
